earthpy/spatial.py

```diff
@@ -197,22 +333,23 @@ def stack_raster_tifs(band_paths, out_path, arr_out=True):
         if arr_out:
             # Write stacked gtif file
             with rio.open(out_path, "w", **dest_kwargs) as dest:
-                stack(sources, dest)
+                _stack(sources, dest)
             # Read and return array
             with rio.open(out_path, "r") as src:
                 return src.read(), src.profile
         else:
             # Write stacked gtif file
             with rio.open(out_path, "w", **dest_kwargs) as dest:
-                return stack(sources, dest)
+                return _stack(sources, dest)
 
 
 # Function to be submitted to rasterio
 # TODO: add unit tests - some are here:
 # https://github.com/mapbox/rasterio/blob/master/rasterio/mask.py
 # This function doesn't stand alone because it writes to an open object called
 # in the other function
-def stack(sources, dest):
+# @deprecate
+def _stack(sources, dest):
```

Comment:
@joemcglinchy im a bit confused here. the  deprecated function was called `stack()` we never had a `_stack`. is something duplicated in this PR by chance?

Reply:
or perhaps it just should be called stack() which i think was the original helper function name

Reply:
stack is now the name of the 'visible' function. stack_raster_tifs used to call stack, which i changed to _stack, since it was a helper method in that context.

For the deprecated methods (stack_raster_tifs and _stack), I'm considering just blowing them away. Thoughts?

Reply:
ooooh i get your issue now @joemcglinchy . in this instance i think you can remove the old version of stack i get it. you have changed stack() not deprecated it. I think we just need to work about deprecating stack_Raster_tifs since that was the original parent function. is that what you mean by blowing stack away? i would still keep stack_Raster_tifs but just have the message saying it's deprecated for anyone who tries to use it. the deprecation message should just say - "hey this is deprecated - use stack() instead" . Does that make sense?